client/src/components/StarRating.jsx

```diff
@@ -1,43 +1,45 @@
+/* eslint-disable camelcase */
 import React from 'react';
+import PropTypes from 'prop-types';
 import StarRatingComponent from 'react-star-rating-component';
+import styles from './style/starrating.css';
 
 class StarRating extends React.Component {
-  constructor() {
-    super();
+  constructor(props) {
+    super(props);
     this.state = {
       rating_half_star: 3.5,
     };
-    this.onStarClickHalfStar = this.onStarClickHalfStar.bind(this);
-  }
-
-  onStarClickHalfStar(e, nextValue, prevValue, name) {
-    const xPos = (e.pageX - e.currentTarget.getBoundingClientRect().left) / e.currentTarget.offsetWidth;
-
-    if (xPos <= 0.5) {
-      nextValue -= 0.5;
-    }
-
-    console.log('name: %s, nextValue: %s, prevValue: %s', name, nextValue, prevValue);
-    this.setState({ rating_half_star: nextValue });
+    // still need to work on this to get it rendering correctly
   }
 
   render() {
+    const { rating_half_star } = this.state;
+    const { rating } = this.props;
+    const avgRating = { rating }.rating[0] || {};
     return (
-      <StarRatingComponent
-        name="overallrate"
-        // eslint-disable-next-line react/destructuring-assignment
-        value={this.state.rating_half_star}
-        starCount={5}
-        starColor="red"
-        emptyStarColor="#f6f6f6"
-        // onStarClick={this.onStarClickHalfStar}
-        // renderStarIcon={(index, value) => (
-        //   <span className={index <= value ? 'fa fa-star' : 'fa fa-star-o'} />
-        // )}
-        // renderStarIconHalf={() => <span className="fa fa-star-half-full" />}
-      />
+      <div className={styles.staroverall}>
+        <StarRatingComponent
+          name="overallrate"
+          value={rating_half_star}
+          starCount={5}
+          starColor="red"
+          emptyStarColor="#f6f6f6"
+          // onStarClick={this.onStarClickHalfStar}
+          // renderStarIcon={(index, value) => (
```

Comment:
Why is this commented out?

Reply:
I couldn't get the half stars working and rendering correctly yet

Reply:
that can be it's own feature branch later. just save the code somewhere and come back to it.